### PR TITLE
first draft of release notes for 3.7.0

### PR DESCRIPTION
**Pull request by kmckeedocs**
Dear friends, you are the only three people available to review the release notes as a pull request. I'll be creating a second review in Google Docs and add last minute changes here.

### CODE REVIEW

**Comment by danilito19** [comment]
@kmckeedocs not seeing references to python updates, should I wait for an updated commit?

**Comment by kmckeedocs** [comment]
Sorry about that- they should be there now, yeah?

KRISTEN McKEE <mailto:kristen.mckee@contrastsecurity.com>  | Senior UX and Documentation Writer
Contrast Security, Inc.<https://www.contrastsecurity.com/>
Welcome to the Era of Self-Protecting Software

File: content/release/ReleaseNotes370.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Agent Updates
+
+## New and Improved Features
+
+PDF Compliance Reports now include the latest Payment Card Industry (PCI) Security Standard version: PCI 3.2.1
+
+### Java Agent Summary
+
+Accuracy improved for Assess and Protect, in particular, data flow accuracy for Java 11 applications. Fixed Assess vulnerability reporting for Mulesoft based applications. Added Route Based Auto Verification (RBAV) support and improved configuration error reporting.
```
Comment (gilday) [inline content/release/ReleaseNotes370.md]: Consider moving the following to `Bug Fixes` section:

> Fixed Assess vulnerability reporting for Mulesoft based applications.

Comment (kmckeedocs) [reply content/release/ReleaseNotes370.md]: Thank you, moved.

File: content/release/ReleaseNotes370.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Accuracy improved for Assess and Protect, in particular, data flow accuracy for Java 11 applications. Added route-based auto-verification support and improved configuration error reporting.
+
+### Node.js Agent Summary 
+
+This release includes several performance improvements and fixes. All customers should upgrade to this version.  
+
+### Python Agent Summary 
+
+Improvements are mostly related to Assess. This includes fixes to sqlite3 patching, support for the Assess SSRF rule, RBAV, reporting, and communication with Contrast Service. The agent no longer sends an XSS vulnerability if the response content type is whitelisted. It can also create an XSS vulnerability outside of rendering a template for django, flask, or werkzeug-compliant frameworks.
```
Comment (gdvdsn-contrast) [inline content/release/ReleaseNotes370.md]: Hi Kristen, here's another instance of RBAV (should be auto-verification)

Comment (kmckeedocs) [reply content/release/ReleaseNotes370.md]: Thanks!

**Comment by danilito19** [comment]
@kmckeedocs sorry for not getting this on time. Now that I know what release notes have to look like, this can be a good example for us to get it right the first time. Thanks!

**Pull request merged**